specification/dynatrace/resource-manager/readme.md

```diff
@@ -37,6 +37,10 @@ These settings apply only when `--tag=package-2023-04-27` is specified on the co
 ```yaml $(tag) == 'package-2023-04-27'
 input-file:
   - Dynatrace.Observability/stable/2023-04-27/dynatrace.json
+suppressions:
+  - code: MISSING_APIS_IN_DEFAULT_TAG
+    from: swagger.yaml
+    reason: API path is not needed anymore.
```

Comment:
Can you clarify in the reason the API is deprecated? From [Teams](https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1685611811702?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1685610617172&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1685611811702) I've read:

> I guess the PR authors have added the suppression for this avocado rule -> MISSING_APIS_IN_DEFAULT_TAG, with the explanation that the API is deprecated and hence its cleaned up, its not being used in the last stable version.


Per the [TSG](https://github.com/Azure/avocado/blob/main/README.md#missing_apis_in_default_tag) we have:

> If the missing API swaggers are deprecated, in this case this error could be ignored.

So a reason that clarifies the API swagger is deprecated clear things up here.

Reply:
@konrad-jamrozik  - Can you check again? I see the readme.md is updated with reason now.  

@kanupriyasinghh @abhargavams